arcgis-ios-sdk-samples/Layers/ArcGIS vector tiled layer (URL)/VectorTileLayerViewController.swift

```diff
@@ -20,59 +20,92 @@ import ArcGIS
 class VectorTileLayerViewController: UIViewController {
 
     @IBOutlet var mapView: AGSMapView!
-    @IBOutlet weak var segmentedControl: UISegmentedControl!
     
-    private var midCenturyURLString = "https://www.arcgis.com/home/item.html?id=7675d44bb1e4428aa2c30a9b68f97822"
-    private var coloredPencilURLString = "https://www.arcgis.com/home/item.html?id=4cf7e1fb9f254dcda9c8fbadb15cf0f8"
-    private var newsPaperURLString = "https://www.arcgis.com/home/item.html?id=dfb04de5f3144a80bc3f9f336228d24a"
-    private var novaURLString = "https://www.arcgis.com/home/item.html?id=75f4dfdff19e445395653121a95a85db"
-    private var nightURLString = "https://www.arcgis.com/home/item.html?id=86f556a2d1fd468181855a35e344567f"
+    /// The model array containing the layer options.
+    private let vectorTiledLayerItems: [(label: String, id: String)] = [
```

Comment:
Should the second parameter label be `identifier`? Or maybe even `itemIdentifier`? I see `id` and I immediately think *pointer to an Objective-C object*.

Reply:
Haha that's what I get for not coming from Objective-C 😂

Reply:
I went with `itemID` in  order to match the corresponding property of `AGSItem`.